tests/testing_support/fixtures.py

```diff
@@ -1483,35 +1483,41 @@ def code_coverage_fixture(source=None):
     if source is None:
         source = ["newrelic"]
 
+    github_actions = bool(os.environ.get("GITHUB_ACTIONS", None))
+    tox_env_directory = os.environ.get("TOX_ENVDIR", None)
+
+    if tox_env_directory:
+        data_file = os.path.join(tox_env_directory, ".coverage")
+        data_suffix = os.path.split(tox_env_directory)[-1]
+        coverage_directory = os.path.join(tox_env_directory, "htmlcov")
+        xml_report = os.path.join(tox_env_directory, "coverage.xml")
+    else:
+        data_file = ".coverage"
+        data_suffix = None
+        coverage_directory = "htmlcov"
+        xml_report = "coverage.xml"
+
     @pytest.fixture(scope="session")
     def _code_coverage_fixture(request):
         if not source:
-            return
-
-        if os.environ.get("GITHUB_ACTIONS") is not None:
+            yield None
```

Comment:
Do you need this `yield None` here or would the `return` below it be sufficient?

Reply:
Since it's a generator based fixture, without the yield None you'll get a 

`ValueError: code_coverage did not yield a value`

I'll put a comment to this effect.